scripts/ci.sh

```diff
@@ -13,12 +13,19 @@ if [ "$TEST" = "unit" ]; then
   npm run test:unit
   npm run dtslint
 
-elif [ "$TEST" = "unit_and_e2e" ]; then
+elif [ "$TEST" = "unit_and_e2e_clients" ]; then
 
   npm run build
-  npm run test:e2e:all
+  npm run test:e2e:ganache
+  npm run test:e2e:geth:insta
+  npm run test:e2e:geth:auto
```

Comment:
Explicitly listing these because if they are grouped together in an `npm script`, CI will pass as long as the last command succeeds. Earlier failures weren't getting detected.